low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1

```diff
@@ -101,6 +101,7 @@ IP_addr=172.16.1.222
 max_login_attempts=2
 
 [RAIDSENSOR]
+should_monitor=true # RAIDSENSOR
```

Comment:
avoid comment for sensor name, its under sensor header already.

same comment elsewhere